contrib/seeds/makeseeds.py

```diff
@@ -46,10 +46,16 @@ def parseline(line: str) -> Union[dict, None]:
     """ Parses a line from `seeds_main.txt` into a dictionary of details for that line.
     or `None`, if the line could not be parsed.
     """
+    if line.startswith('#'):
```

Comment:
> It looks like this will never be true, unless a user manually adds # characters

The first line seeds_main.txt is # addresses, so it's going to be hit at least once.

Reply:
It catches the first line for me.

Reply:
Yes, i've meant to ignore the first line (the comment for column names) in  `seeds_main.txt` by putting the **"#"** conditional.

@jonatack I've thought about comparing it in **str** but then decided to cast with **int** because, seems like the `good` column meant to be **0 and 1** which is **integer**, so i rather use casting and have the **"#"** conditional.